docs/reference/stdlibs/std/chain.md

```diff
@@ -117,8 +134,24 @@ currentRealm := std.CurrentRealm()
 ```go
 func PrevRealm() Realm
 ```
-Returns the previous caller [realm](realm.md) (can be code or user realm). If caller is a
-user realm, `pkgpath` will be empty.
+
+PrevRealm returns the [Realm](realm.md) which called the code being executed.
+The value of PrevRealm remains the same if called by a function within the
+same realm, or a function in a pure package. It will change if called by a
```

Comment:
> It will change if called by a realm with a different pkgpath.

Was this copied from the PrevRealm section? Because it doesn't change when called by realm1 which has a different pkgpath than the main package.